ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Overview/CVEsTableContainer.tsx

```diff
@@ -51,17 +61,56 @@ function CVEsTableContainer({
 
     const { data: imageCountData } = useQuery(unfilteredImageCountQuery);
 
+    const selectedCves = useSet<string>();
+    const [exceptionRequestModalOptions, setExceptionRequestModalOptions] =
+        useState<ExceptionRequestModalOptions>(null);
+
+    function openDeferralModal() {
+        setExceptionRequestModalOptions({ type: 'DEFERRAL', cves: selectedCves.asArray() });
+    }
+
+    function openFalsePositiveModal() {
+        setExceptionRequestModalOptions({ type: 'FALSE_POSITIVE', cves: selectedCves.asArray() });
+    }
+
     const tableData = data ?? previousData;
     return (
         <>
+            {exceptionRequestModalOptions && (
+                <ExceptionRequestModal
+                    cves={exceptionRequestModalOptions.cves}
+                    type={exceptionRequestModalOptions.type}
+                    onClose={() => setExceptionRequestModalOptions(null)}
+                />
+            )}
             <TableEntityToolbar
                 defaultFilters={defaultFilters}
                 countsData={countsData}
                 setSortOption={setSortOption}
                 pagination={pagination}
                 tableRowCount={countsData.imageCVECount}
                 isFiltered={isFiltered}
-            />
+            >
+                <ToolbarItem alignment={{ default: 'alignRight' }}>
+                    <BulkActionsDropdown isDisabled={selectedCves.size === 0}>
+                        <DropdownItem
+                            key="bulk-defer-cve"
+                            component="button"
+                            onClick={openDeferralModal}
+                        >
+                            Defer CVEs
+                        </DropdownItem>
+                        <DropdownItem
+                            key="bulk-mark-false-positive"
+                            component="button"
+                            onClick={openFalsePositiveModal}
+                        >
+                            Mark as false positives
```

Comment:
Is the meaning **Mark CVEs as false positive**?

Maybe a quirk of my mind that it stops to reread with what feels like missing direct object.